README.md

```diff
@@ -2,7 +2,7 @@
 
 A Ruby library to expand shortened URLs.
 
-**Current version:** 1.1.0  
+**Current version:** 1.2.0
```

Comment:
Can we keep the two trailing spaces here as they're required for new lines in Markdown?